streams/src/main/java/org/apache/kafka/streams/AutoOffsetReset.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.streams;
+
+public class AutoOffsetReset {
```

Comment:
nit: kind of confusing since we have an enum of the same name in the `Topology` class (not to mention it's empty -- guessing it will be filled in with/after the other PR?)

Reply:
What do you mean by "it's empty" -- the KIP does actually deprecate `Topology.AutoOffsetReset` in favor of this new class (we need a class, as we need to not just track the strategy itself, but for `by_duration` the additional reset duration

cf the other PR

Reply:
sorry, should have looked at the KIP and the other PR first 😞 shame on me